index.html

```diff
@@ -5,7 +5,7 @@
    <title>Salmon Cookies Are Delicious</title>
```

Comment:
on lines 8 and 15, remove space between attributes and =. Should be id=', src=', and type='. In HTML, there are no spaces between the =.